### ISSUE TITLE
Unhandled Exception Crash

### ISSUE THREAD

**Issue by chgi** [rmcrackan/Libation#492]
Hello there, I just found this project and tried it out. Great stuff! Thank you!

**Describe the bug**
I got a crash that may be related to non-threadsafe behaviour in Avalonia.
This didn't leave an entry in the log file, but I got the output below on the console.
After a little searching, I think it might be related to to <https://github.com/AvaloniaUI/Avalonia/issues/8810>.
I am not familiar with Avalonia, but it seems to me this could be fixed by 
dispatching `RunningTime = timeToStr(DateTime.Now - StartingTime);` to the UI thread in `Source/LibationAvalonia/ViewModels/ProcessQueueViewModel.cs:256` instead of updating it directly from the timer thread.

```
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at Avalonia.Utilities.WeakHashList`1.GetAlive(Func`1 factory) in /_/src/Avalonia.Base/Utilities/WeakHashList.cs:line 239
   at Avalonia.Utilities.WeakEvent`2.Subscription.OnEvent(Object sender, TEventArgs eventArgs) in /_/src/Avalonia.Base/Utilities/WeakEvent.cs:line 157
   at System.Reactive.AnonymousSafeObserver`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/AnonymousSafeObserver.cs:line 54
   at System.Reactive.Linq.ObservableImpl.SelectMany`2.EnumerableSelector._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/SelectMany.cs:line 1379
   at System.Reactive.Linq.ObservableImpl.Buffer`2.Boundaries._.BufferClosingObserver.OnNext(TBufferClosing value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Buffer.cs:line 829
   at System.Reactive.Linq.ObservableImpl.Merge`1.Observables._.InnerObserver.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Merge.cs:line 240
   at System.Reactive.Linq.ObservableImpl.Select`2.Selector._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Select.cs:line 48
   at System.Reactive.Subjects.Subject`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/Subjects/Subject.cs:line 141
   at ReactiveUI.IReactiveObjectExtensions.ExtensionState`1.RaisePropertyChanged(String propertyName) in /_/src/ReactiveUI/ReactiveObject/IReactiveObjectExtensions.cs:line 397
   at ReactiveUI.IReactiveObjectExtensions.RaisingPropertyChanged[TSender](TSender reactiveObject, String propertyName) in /_/src/ReactiveUI/ReactiveObject/IReactiveObjectExtensions.cs:line 243
   at LibationAvalonia.ViewModels.ProcessQueueViewModel.CounterTimer_Tick(Object state) in /home/chris/libationBuild/src/Libation-9.3.0/Source/LibationAvalonia/ViewModels/ProcessQueueViewModel.cs:line 257
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.TimerQueueTimer.Fire(Boolean isThreadPool)
   at System.Threading.TimerQueue.FireNextTimers()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
   at System.Threading.PortableThreadPool.WorkerThread.WorkerThreadStart()
zsh: IOT instruction (core dumped)  libation
```

**To Reproduce**
Not sure. Happend a little after I hit the retry button on a timed out download, but looking at the code, it is pretty obvious that it is not reliably reproducible.

**Platform**
Libation 9.3.0
Manjaro Linux

**Comment by Mbucari** [comment]
@chgi thanks for the sleuthing! I think I've encountered this a couple of times before, but I wasn't able to reproduce it so I just ignored it. Is this an occasional crash for you, or does it nearly always happen?

**Comment by chgi** [comment]
@Mbucari I think I had 2 crashes in 160 audio books so far. One of those got me stack trace above.

**Comment by rmcrackan** [comment]
Fixed by @Mbucari in [version 9.3.1](https://github.com/rmcrackan/Libation/releases/latest)

**Comment by rmcrackan** [comment]
Also addressed in latest -- [v9.3.3](https://github.com/rmcrackan/Libation/releases/latest)
